pkg/utils/distro/default_distro.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build !distro
+
+package distro
+
+var DistroYAML = []byte("tidb: TiDB\ntikv: TiKV\ntiflash: TiFlash\npd: PD\n")
```

Comment:
This is hard to maintain when we want to add more distro variables. Could you improve it?

Reply:
Yes. I got a good idea that maybe I can generate the distro instance direct.